lib/std/crypto.zig

```diff
@@ -13,6 +13,8 @@ pub const aead = struct {
     pub const XChaCha20Poly1305 = chacha20.XChacha20Poly1305;
     pub const AEGIS128L = @import("crypto/aegis.zig").AEGIS128L;
     pub const AEGIS256 = @import("crypto/aegis.zig").AEGIS256;
+    pub const AES128GCM = @import("crypto/aes_gcm.zig").AES128GCM;
+    pub const AES256GCM = @import("crypto/aes_gcm.zig").AES256GCM;
```

Comment:
Consider renaming this to Aes128GCM or Aes256Gcm cause all caps is illegible

Reply:
Nod. But `AES` is written `AES` in other files, so I’d rather keep things consistent in that PR, and make a global renaming (we have other identifiers in all caps, such as AEGIS) in a separate PR.